Fix/cleanup readme

mikeyarce opened the pull request:
Partially Fixes: https://github.com/Parsely/wp-parsely/issues/186

I think the main remaining task is to add a plugin banner cc @dabowman

What this does:
- Updates screenshots
- Moves from code screenshots to code snippets
- Improves installation instructions

mikeyarce committed:
Refresh screenshots and update readme

Refresh screenshots with more recent higher resolution images.  Update readme with new screenshots.

mikeyarce committed:
Add sample metadata section to Readme

Since we moved the sample metadata out of the screenshots, I added a section that includes those samples.

mikeyarce committed:
Remove redundant FAQ

In question 1 we already mention the free trial offer, we don’t need to do that again.

mikeyarce committed:
Update Installation instructions

Include both manual and UI instructions, added a paragraph above that talks about requiring a Parse.ly account and how to get a free trial and a bit about what Parse.ly does.

mikeyarce committed:
Fixed version, body tag, and added semi-colons

mikeyarce committed:
Change screenshot alt text to be more descriptive

mikeyarce committed:
Optimized screenshot images

Used `ImageOptim`
`Saved 581 KB out of 1.2 MB. 47.2% per file on average (up to 48.4%)`

GaryJones commented on README.md:
```diff
@@ -28,14 +28,22 @@ Feedback, suggestions, questions or concerns? E-mail us at [support@parsely.com]
-1. Head to the settings page for the plugin (should be /wp-admin/options-general.php?page=parsely)
-1. Set your Site ID, which is your own site domain name (e.g., `mysite.com`)
-1. Save your changes and enjoy your data!
+The WP-Parsely plugin requires an active Parse.ly account. Parse.ly gives creators, marketers and developers the tools to understand content performance, prove content value, and deliver tailored content experiences that drive meaningful results.
```
```suggestion
The plugin requires an active Parse.ly account. Parse.ly gives creators, marketers, and developers the tools to understand content performance, prove content value, and deliver tailored content experiences that drive meaningful results.
```

mikeyarce replied:
This was taken from the main Parse.ly homepage - but I agree this comma is needed.

GaryJones commented on README.md:
```diff
@@ -79,17 +87,39 @@ See [the wiki](https://github.com/Parsely/wp-parsely/wiki/Setting-up-a-WP-plugin
 ## Screenshots
 
-1. The main settings screen of the wp-parsely plugin  
+1. The main settings screen of the WP-Parsely plugin  
```
The plugin isn't called `WP-Parsely`. It's called `Parse.ly`, and has a slug of `wp-parsely`. Let's aim not to mix the two.
```suggestion
1. The main settings screen of the Parse.ly plugin.  
```

GaryJones commented on README.md:
```diff
@@ -79,17 +87,39 @@ See [the wiki](https://github.com/Parsely/wp-parsely/wiki/Setting-up-a-WP-plugin
+
+## Sample Parse.ly metadata
+
+The standard Parse.ly JavaScript tracker inserted before the closing `body` tag:
+
+    <!-- START Parse.ly Include: Standard -->
+
+       <script data-cfasync="false" id="parsely-cfg" data-parsely-site="wpvip.com" src="//cdn.parsely.com/keys/wpvip.com/p.js"></script>
+
+    <!-- END Parse.ly Include: Standard -->
```
This bit isn't the metadata, so it should be moved outside of the `Sample Parse.ly metadata` section (can have its own heading).

Also:

```suggestion
The standard Parse.ly JavaScript tracker inserted before the closing `body` tag:

    <!-- START Parse.ly Include: Standard -->

       <script data-cfasync="false" id="parsely-cfg" data-parsely-site="example.com" src="https://cdn.parsely.com/keys/example.com/p.js"></script>

    <!-- END Parse.ly Include: Standard -->
```

GaryJones commented on README.md:
```diff
@@ -79,17 +87,39 @@ See [the wiki](https://github.com/Parsely/wp-parsely/wiki/Setting-up-a-WP-plugin
+
+    <!-- END Parse.ly Include: Standard -->
+
+A sample `JSON-LD` meta tag for a home page or section page:
```
```suggestion
A sample `JSON-LD` meta tag and structured data for a home page or section page:
```

Really, the metatag with the plugin version is separate from the JSON-LD. We may be able to remove it.

GaryJones commented on README.md:
```diff
@@ -79,17 +87,39 @@ See [the wiki](https://github.com/Parsely/wp-parsely/wiki/Setting-up-a-WP-plugin
 
-3. A sample `JSON-LD` meta tag for a home page or section page
-![3. A sample `JSON-LD` meta tag for a home page or section page](.wordpress-org/screenshot-3.png)
+A sample `JSON-LD` meta tag for an article or post:
```
```suggestion
A sample `JSON-LD` structured data element for an article or post:
```

mikeyarce committed:
Update README.md

Add comma

Co-authored-by: Gary Jones <gary.jones@automattic.com>

mikeyarce committed:
Update README.md

Fix plugin name

Co-authored-by: Gary Jones <gary.jones@automattic.com>

mikeyarce committed:
Use example domain instead of wpvip.com and add url scheme

Co-authored-by: Gary Jones <gary.jones@automattic.com>

mikeyarce committed:
Add clarification to code samples

GaryJones committed:
Docs: Update readme

- More specific images, and more optimized for file size.
- Fix up Tags, grammar, paragraphs, extra words, code snippets, and add other clarifications.
